src/components/datePicker/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ChangeEventHandler, FC, FocusEventHandler } from 'react';
+import { Input } from '@chakra-ui/react';
+
+type Props = {
+  onBlur?: FocusEventHandler<HTMLInputElement>;
+  onFocus?: FocusEventHandler<HTMLInputElement>;
+  onChange?: ChangeEventHandler<HTMLInputElement>;
+  size?: 'md' | 'lg';
+  min?: Date;
+  value?: string;
+};
```

Comment:
```suggestion
} & Pick<InputProps, 'onFocus' | 'onBlur' | 'onChange'>;;
```

I would go for this and not repeat the event handlers to avoid mistyping them